webrtc.html

```diff
@@ -2363,11 +2363,14 @@
                 which point the <code><a>RTCPeerConnection</a></code> can fully
                 adopt the pending local description, or rollback to the current
                 description if the remote side rejected the change.</p>
+                <p>As noted in <span data-jsep="ice-candidate-pool">[[!JSEP]]</span>,
+                calling this method triggers the ICE candidate gathering process
+                by the <a>ICE Agent</a>.</p>
                 <p>As noted in <span data-jsep="modifying-sdp">[[!JSEP]]</span>
                 the SDP returned from <code>createOffer</code> or
                 <code>createAnswer</code> MUST NOT be changed
-                before passing it to <code>setLocalDescription</code>. As
```

Comment:
There are now two notes referencing JSEP, and I wanted to split it into multiple paragraphs to improve readability. I see the steps are defined the same way as other methods, so it's not like there are steps in `setLocalDescription` because of the JSEP requirement.

Anyway I have added back the "as a result", since it doesn't matter much either way.